server/routes/activities/manage-allocations/handlers/startJourney.ts

```diff
@@ -56,11 +56,15 @@ export default class StartJourneyRoutes {
       updatedExclusions: [],
     }
 
+    if (!inmate.activeFlag) {
```

Comment:
I've tested by suspending from the OCUCANTR screen and this seems to work, but perhaps there's something I'm missing about this flag? I'm worried there might be edge cases here – could a prisoners activities be suspended, but be active in another prison, for example?

Reply:
Yes - it's possible they will have been transferred - so still active, but not in the prison where their allocations were.

Reply:
Updated logic to check prisoner's current prison also matches the user's active case load.